test/javascript/components/journey/SolutionsList.test.js

```diff
@@ -0,0 +1,205 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+import '@testing-library/jest-dom/extend-expect'
+import { SolutionsList } from '../../../../app/javascript/components/journey/SolutionsList'
+import userEvent from '@testing-library/user-event'
+
+test('pulls solutions', async () => {
+  const solutions = [
+    {
+      id: 'uuid-1',
+      exerciseTitle: 'Lasagna',
+    },
+    {
+      id: 'uuid-2',
+      exerciseTitle: 'Bob',
+    },
+  ]
+
+  const server = setupServer(
+    rest.get('https://exercism.test/solutions', (req, res, ctx) => {
+      return res(
+        ctx.status(200),
+        ctx.json({
+          results: solutions,
+          meta: {
+            current: 1,
+            total: 1,
+          },
+        })
+      )
+    })
+  )
+  server.listen()
+
+  render(<SolutionsList endpoint="https://exercism.test/solutions" />)
+
+  expect(await screen.findByText('Showing 2 solutions')).toBeInTheDocument()
+  expect(await screen.findByText('Lasagna')).toBeInTheDocument()
+  expect(await screen.findByText('Bob')).toBeInTheDocument()
+
+  server.close()
+})
+test('paginates solutions', async () => {
```

Comment:
A little visual separation makes it easier to identify the different tests I feel

```suggestion

test('paginates solutions', async () => {
```